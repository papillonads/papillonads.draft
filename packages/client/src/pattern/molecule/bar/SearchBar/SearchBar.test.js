import searchBarDefault from './SearchBar.render'

describe('<SearchBar />', () => {
  describe('Snaphot', () => {
    test('must match searchBarDefault', () => {
      expect(global.renderToJSON(searchBarDefault)).toMatchSnapshot()
    })
  })
})
