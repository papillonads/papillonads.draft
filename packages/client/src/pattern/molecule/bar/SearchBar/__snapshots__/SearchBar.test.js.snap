// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> Snaphot must match searchBarDefault 1`] = `null`;
